vj4/ui/components/ide/ide.styl

```diff
@@ -0,0 +1,279 @@
+        background: #efefef
+      .result__progress-bg
+        position: absolute
+        background: $record-status-color.progress
+        opacity: 0.15
+        height: 100%
+        transition: width 0.2s ease-in-cubic
+      .result__title
+        display: inline-block
+        width: 100px
+        padding-left: 5px
+        i
+          padding: 5px
+      .result__title__text--pretest
+        color: $small-color !important
+      .result__state
+        display: inline-block
+        min-width: 200px
+      .result__state__items
+        display: inline-block
+      .result__state__items__item
+        display: inline-block
+        padding: 0 5px
+      .result__state__items__item--pass
+        color: $record-status-color.pass !important
+      .result__state__items__item--fail
+        color: $record-status-color.fail !important
+      .result__state__items__item--normal
+        color: $small-color !important
+      .result__statistics
+        display: inline-block
+        float: right
+      .result__statistics__running-time, .result__statistics__memory-usage, .result__statistics__start-time
+        display: inline-block
+        padding: 0 5px
+        line-height: 2.1875rem
+        color: $small-color
+
+    .result--pending
+      border-left: 5px solid $record-status-color.pending
+      .result__title__icon
+        &:before
+          content: $record-status-icon.pending
+        color: $record-status-color.pending
+      .result__title__text, .result__state__items__item
+        color: $record-status-color.pending
+    .result--progress
+      border-left: 5px solid $record-status-color.progress
+      .result__title__icon
+        &:before
+          content: $record-status-icon.progress
+        color: $record-status-color.progress
+      .result__title__text, .result__state__items__item
+        color: $record-status-color.progress
+    .result--pass
+      border-left: 5px solid $record-status-color.pass
+      .result__title__icon
+        &:before
+          content: $record-status-icon.pass
+        color: $record-status-color.pass
+      .result__title__text, .result__state__items__item
+        color: $record-status-color.pass
+    .result--fail
+      border-left: 5px solid $record-status-color.fail
+      .result__title__icon
+        &:before
+          content: $record-status-icon.fail
+        color: $record-status-color.fail
+      .result__title__text, .result__state__items__item
+        color: $record-status-color.fail
+    .result--ignore
+      border-left: 5px solid $record-status-color.ignored
+      .result__title__icon
+        &:before
+          content: $record-status-icon.ignored
+        color: $record-status-color.ignored
+      .result__title__text, .result__state__items__item
+        color: $record-status-color.ignored
+
+
+  /* CodeMirror style fixes */
+  .ReactCodeMirror
+    display: table-row
+    height: 100%
+    min-height: 0
+    font-family: $code-font-family
+  .CodeMirror
```

Comment:
Avoid overrides `CodeMirror` styles globally, otherwise it affects `simple-markdown-editor`